tests/simulate/test_lammps.py

```diff
@@ -0,0 +1,146 @@
+import tempfile
+import unittest
+
+import lammps
+import numpy as np
+
+import relentless
+from ..potential.test_pair import LinPot
+
+class test_LAMMPS(unittest.TestCase):
+    """Unit tests for relentless.LAMMPS"""
+
+    def setUp(self):
+        self._tmp = tempfile.TemporaryDirectory()
+        self.directory = relentless.Directory(self._tmp.name)
+
+    #mock (NVT) ensemble and potential for testing
+    def ens_pot(self):
+        ens = relentless.Ensemble(T=2.0, V=relentless.Cube(L=10.0), N={'1':2,'2':3})
+        ens.P = 2.5
+
+        # setup potentials
+        pot = LinPot(ens.types,params=('m',))
+        for pair in pot.coeff:
+            pot.coeff[pair]['m'] = 2.0
+        pots = relentless.simulate.Potentials()
+        pots.pair.potentials.append(pot)
+        pots.pair.rmax = 10.0
+        pots.pair.num = 11
+
+        return (ens,pots)
+
+    def create_file(self):
+        file_ = self.directory.file('test.data')
+        with open(file_,'w') as f:
+            f.write(('LAMMPS test data\n'
+                     '\n'
+                     '5 atoms\n'
+                     '2 atom types\n'
+                     '\n'
+                     '0.0 10.0 xlo xhi\n'
+                     '0.0 10.0 ylo yhi\n'
+                     '0.0 10.0 zlo zhi\n'
+                     '\n'
+                     'Atoms\n'
+                     '\n'
+                     '1 1 1.0 1.0 1.0\n'
+                     '2 1 3.0 3.0 3.0\n'
+                     '3 2 5.0 5.0 5.0\n'
+                     '4 2 7.0 7.0 7.0\n'
+                     '5 2 9.0 9.0 9.0\n'
+                     '\n'
+                     'Masses\n'
+                     '\n'
+                     '1 0.3\n'
+                     '2 0.1'))
+        return file_
+
+    def test_initialize(self):
+        """Test running initialization simulation operations."""
+        #InitializeFromFile
+        ens,pot = self.ens_pot()
+        file_ = self.create_file()
+        op = relentless.simulate.lammps.InitializeFromFile(filename=file_, neighbor_buffer=0.4)
+        l = relentless.simulate.lammps.LAMMPS(operations=op, quiet=False)
+        sim = l.run(ensemble=ens, potentials=pot, directory=self.directory)
+
+        #InitializeRandomly
+        ens,pot = self.ens_pot()
+        op = relentless.simulate.lammps.InitializeRandomly(neighbor_buffer=0.4, seed=1)
+        l = relentless.simulate.lammps.LAMMPS(operations=op, quiet=False)
+        sim = l.run(ensemble=ens, potentials=pot, directory=self.directory)
+
+    def test_minimization(self):
+        """Test running energy minimization simulation operation."""
+        #MinimizeEnergy
+        ens,pot = self.ens_pot()
+        file_ = self.create_file()
+        op = [relentless.simulate.lammps.InitializeFromFile(filename=file_, neighbor_buffer=0.4),
+              relentless.simulate.lammps.AddNVTIntegrator(dt=0.5,tau_T=1.0),
+              relentless.simulate.lammps.MinimizeEnergy(energy_tolerance=1e-7,
```

Comment:
You don't need (or want) the NVT integrator here because MinimizeEnergy executes its own run. You can AddNVTIntegrator and Run after the minimization, or in a separate test.